core/tests/puppeteer-acceptance-tests/utilities/user/exploration-editor.ts

```diff
@@ -960,6 +960,25 @@ export class ExplorationEditor extends BaseUser {
       throw new Error(`User ${username} is not a subscriber.`);
     }
   }
+
+  async playExploration(explorationId): Promise<void> {
+    await this.page.goto(`${baseURL}/explore/${explorationId}`);
+    await this.page.waitForNavigation({waitUntil: ['load']});
+  }
+
+  async giveFeedback(feedback: string): Promise<void> {
+    await this.clickOn('.e2e-test-exploration-feedback-popup-link');
```

Comment:
Please store this selector string as a constant. Ditto for the one below.

Reply:
done.